docs/source/cpp/compute.rst

```diff
@@ -665,6 +665,57 @@ String component extraction
   e.g. 'letter' and 'digit' for the regular expression
   ``(?P<letter>[ab])(?P<digit>\\d)``.
 
+Temporal component extraction
```

Comment:
Nit: can you move these below other string-related sections?

Reply:
Moved. Please check if the location is ok.